tests/package-custom-coverage-test.js

```diff
@@ -234,9 +234,9 @@ describeApplication('PackageCustomCoverage', () => {
           });
 
           it('rejects invalid begin date', () => {
-            convergeOn(() => {
+            return new Convergence(() => {
               expect(PackageShowPage.validationError).to.exist;
-            }).then(() => {
+            }).do(() => {
```

Comment:
`.do()` is used to run an arbitrary function _after_ a convergence. This should be `.once()` so we can converge on it if it fails. We can also group expectations in a single `.once()`, so we shouldn't have to do `.once().once()` unless you're testing for a specific order of events to happen.

Reply:
Also, `expect(PackageShowPage.validationError).to.exist;` will always pass because `validationError` will always be `true` or `false`. So probably not necessary.

Reply:
Yeah, while debugging this test it doesn't seem like anything ever runs.. Nothing logs out. I'll come back to it on Monday